neat/tools/fittools/compartmentfitter.py

```diff
@@ -375,6 +375,9 @@ def __init__(self, phys_tree,
         self.name = name
         self.path = path
 
+        # boolean flag that is initiallized the first time `self.fitPassive` is called
```

Comment:
why is this variable not initialized with a reasonable default here? it can still be overwritten later?

Reply:
set to `True`